### ISSUE TITLE
[HUD custom font]: the custom font makes UIs inaccessible for couple seconds

### ISSUE THREAD

**Issue by legomushroom** [Glimpse/Glimpse.Client.Hud#147]
Since browser spends some time `downloading`, `parsing` and `rendering` custom font, the text on the HUD is not visible for couple seconds:

![hud-font-issue](https://user-images.githubusercontent.com/1478800/27702053-69215f76-5cc8-11e7-9a36-7aa949d01089.gif)

Need to do better job with custom font loading or get rid of it entirely.

**Comment by davidkpiano** [comment]
Downloading from where? It should be locally served, no?

**Comment by legomushroom** [comment]
@davidkpiano yeah locally but from the web server, anyways the custom font rendered as the empty placeholder until browser finishes all the custom font work.

**Comment by davidkpiano** [comment]
cc. @chetsangnil I talked about this with him a while back (re. using system fonts instead of custom fonts)

**Comment by legomushroom** [comment]
I'd love to use a system font here.

**Comment by mike-kaufman** [comment]
@legomushroom - 
  1.  I haven't observed this behavior on win/chrome.  Can you add repro steps (i.e., what URL are you hitting to see this) & browser/os info?
  2.  How do you know the delay here is related to custom fonts?

**Comment by legomushroom** [comment]
1. I think the repro is:
  - Launch `express.test.app`
  - Goto `Timeline`, click on `Http Services`

2. This is the common problem with custom fonts, as you can see on the gif that everything besides text rendered ok, but the text is missed.
